fitting/Utils.py

```diff
@@ -532,13 +554,13 @@ def checkSBFit(filename,label,roobins,plotname, nPars, plot_dir):
     #chi2,ndof = calculateChi2(hpull, nPars +1)
 
     pdf_names = ["model_s"] 
-    PlotFitResults(frame,fres.GetName(),nPars+1,frame3,"data_obs", pdf_names,chi2,ndof,'sbFit_'+plotname, plot_dir, has_sig = True)
+    PlotFitResults(frame,fres.GetName(),nPars+1,frame3,"data_obs", pdf_names,chi2,ndof,'sbFit_'+plotname, plot_dir, has_sig = True, plot_label = plot_label)
 
     print "chi2,ndof are", chi2, ndof
     return chi2, ndof
 
 
-def f_test(nParams, nDof, chi2, thresh = 0.05):
+def f_test(nParams, nDof, chi2, fit_errs, thresh = 0.05, err_thresh = 0.3 ):
```

Comment:
The default command line argument for `err_thresh` is 0.5. Let's not introduce default values in more than one place, or at the very least keep them consistent.

Reply:
Ah yeah good catch. I'll change the default of this function to match at least. I want to keep a default to not break anyone else's who uses this function